### PR TITLE
add max() for Int and Float

### PR DESCRIPTION
**Pull request by bsnyder788**

### CODE REVIEW

File: gleam_stdlib/test/gleam/float_test.gleam [hunk 1]
```diff
@@ -137,3 +137,26 @@ pub fn truncate_test() {
   |> float.truncate
   |> expect.equal(_, -7)
 }
+
+pub fn max_test() {
+    float.max(0., 0.)
```
Comment (lpil) [inline gleam_stdlib/test/gleam/float_test.gleam]: Can you indent with 2 spaces here and in the int test file please? Thank you

Comment (bsnyder788) [reply gleam_stdlib/test/gleam/float_test.gleam]: done!

**Comment by lpil** [comment]
Sorry, this has merge conflicts now that I've merged the other PR

**Comment by bsnyder788** [comment]
@lpil conflicts should be resolved now

**Pull request merged**